corehq/apps/registration/forms.py

```diff
@@ -46,9 +46,6 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
     """
     Form for a brand new user, before they've created a domain or done anything on CommCare HQ.
     """
-    full_name = forms.CharField(label=_('Full Name'),
```

Comment:
Update: we need to put first name and last name back in there, as optional
fields. Jenny, happy to give context over voice, but basically hubspot has
some compelling evidence that having names helps significantly with
marketing campaigns.

On Mon, Sep 14, 2015 at 4:02 PM, Amelia Sagoff <asagoff@dimagi.com> wrote:

> hey - this is now stalling on some objections from marketing - we may need
> to put firstname/lastname back in there. Will update you when this thread
> is resolved (hopefully soon)
>
> On Thu, Sep 10, 2015 at 10:29 AM, Jonathan Emord <notifications@github.com
> > wrote:
>
>> In corehq/apps/registration/forms.py
>> <https://github.com/dimagi/commcare-hq/pull/8202#discussion_r39166190>:
>>
>> > @@ -46,9 +46,6 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
>> >      """
>> >      Form for a brand new user, before they've created a domain or done anything on CommCare HQ.
>> >      """
>> > -    full_name = forms.CharField(label=_('Full Name'),
>>
>> Ha. I recently used full_name for interrupt
>> https://github.com/dimagi/commcare-hq/pull/8109/files
>>
>> Just realizing now that keeping the username kwarg was probably not the
>> most clear idea
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/dimagi/commcare-hq/pull/8202/files#r39166190>.
>>
>
>